teammapper-frontend/src/app/root.component.ts

```diff
@@ -40,6 +40,15 @@ export class RootComponent implements OnInit {
         replaceUrl: true,
       });
     }
+
+    // Fix for #347: Force a reload of mindmaps, even when they are persisted in bfcache
```

Comment:
please make this comment a bit more descriptive. It's already obvious that a reload happens, the question is why ;) e.g. "Force reload to prevent broken sync state on Mac safari where the URL and the internal state don't match"